pages/quiz.html

```diff
@@ -0,0 +1,333 @@
+          </div>
+
+          <div
+            class="shadow-lg rounded flex flex-col relative w-72 mb-12 ml-12 mr-5"
+          >
+            <div class="h-72">
+              <img
+                class="img-responsive rounded"
+                style="height: 100%"
+                src="https://picsum.photos/200"
+                alt="dress"
+              />
+              <div class="px-4 py-2 text-badge card-badge bg-indigo-300">
+                new
+              </div>
+            </div>
+
+            <div class="p-3">
+              <h3 class="text-grey-600 subtitle1 mb-2">Quiz title</h3>
+
+              <div class="flex justify-between">
+                <a
+                  href=""
+                  class="btn btn-square-solid btn-xs bg-indigo-500 text-white mr-3"
+                >
+                  play quiz
+                </a>
+                <a
+                  id="modal-btn"
+                  class="btn btn-square-solid btn-xs bg-indigo-500 text-white"
+                >
+                  quiz rules
+                </a>
+
+                <div id="modal" class="modal p-20">
+                  <div class="modal-cont bg-grey-400">
+                    <span
+                      class="modal-close-btn close-btn text-slate-700 text-hover-white fas fa-times"
+                    ></span>
+                    <h2>quiz rules</h2>
+                    <p>
+                      Lorem, ipsum dolor sit amet consectetur adipisicing elit.
+                      Fugiat, veniam amet vel iure nulla expedita quisquam nisi
+                      neque repellendus temporibus consectetur eaque fugit
+                      asperiores aut dicta iusto aspernatur ipsum libero?
+                    </p>
+                  </div>
+                </div>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+    </main>
+    <!-- main section -->
+
+    <div class="spacer-3rem"></div>
+    <div class="spacer-3rem"></div>
+    <div class="spacer-3rem"></div>
+    <div class="spacer-3rem"></div>
+
+    <!-- footer -->
+    <footer
+      class="relative flex flex-col h-44 justify-center align-items-center w-full p-5 z-50 bg-cyan-200"
+      style="bottom: 0"
+    >
+      <div class="text-grey-700">
+        Made with
+        <span class="text-cyan-700 font-bold ml-1 mr-1">&lt;/&gt;</span> by
+
+        <a class="ml-1 text-hover-cyan-600">Ananya</a>
+      </div>
+      <div class="flex mt-3 align-items-center justify-center">
+        <a href="https://github.com/Ananyamadhu08" target="_blank">
+          <i class="fab fa-github mr-2 text-cyan-700"></i>
+        </a>
+        <a href="https://twitter.com/AnanyaMadhu27" target="_blank">
+          <i class="fab fa-twitter mr-2 text-cyan-700"></i>
+        </a>
+        <a
+          href="https://www.linkedin.com/in/ananya-madhu-74479b206/"
+          target="_blank"
+        >
+          <i class="fab fa-linkedin mr-2 text-cyan-700"></i>
+        </a>
```

Comment:
It is advised to use rel="noopener noreferrer"  with target="_blank" to improve security.